pages/auth/login.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { getCsrfToken } from "next-auth/react";
+import LayoutSytle from "../../components/LayoutStyle";
+import styled from "styled-components";
+import github from "../../public/images/github.svg";
+import Image from "next/image";
+
+export default function Login({ csrfToken }) {
+  return (
+    <LayoutSytle>
+      <Form action="/api/auth/signin/github" method="POST">
+        <input type="hidden" name="csrfToken" value={csrfToken} />
+        <input type="hidden" name="callbackUrl" value="/" />
```

Comment:
Im sorry to say that this is a bit hacky. You can use a onSubmit handler with event.preventDefault which performs a fetch to the specified api route as a "POST", in the body you can then specify the relevant variables crsfToken and callbackUrl.